src/util/IncognitoBrowser.cpp

```diff
@@ -45,25 +52,71 @@ QString injectPrivateSwitch(QString command)
     return QString();
 }
 
+QString callAssocQueryString(ASSOCF flags, ASSOCSTR str, LPCWSTR pszAssoc,
```

Comment:
Could we move this function into `src/util/WindowsHelper.hpp/cpp` with an "as-minimal-as-possible" wrapper?
By as-minimal-as-possible I mean don't have the users of the function (`getCommand` in this case) pass in any arguments that we always set to null.
If possible, rename the function to something that describes its intent more clearly

Reply:
There is a flag that callers could pass in that would require them to pass in something to pszExtra, related to looking up programs by exe name. I tried to leave in the most flexibility possible for this wrapper, so I didn't prevent the caller from passing in that flag (and therefore left pszExtra to the caller.)

On second thought, it's pretty unlikely that chatterino would need to use that lookup, so I'll make the wrapper much more specific and only support protocol and file extension lookups.